### ISSUE TITLE
Feature request - Duration interface

### ISSUE THREAD

**Issue by santhosh-sk** [directus/app#2977]
It would be great to have a "Duration" field type (Interface), as this is a common use case. Current work around for this is to store it in ISO 8601 duration format, with some custom validation using regular expressions. 

An built in solution with an in intuitive UI would be a better fit for this use case.

**Comment by rijkvanzanten** [comment]
Are you talking about this format specifically? `PnYnMnDTnHnMnS` (for example `P3Y6M4DT12H30M5S`)

**Comment by santhosh-sk** [comment]
> Are you talking about this format specifically? `PnYnMnDTnHnMnS` (for example `P3Y6M4DT12H30M5S`)

Yes, that is one way of storing data. But I would like to emphasize the UI. An intuitive approach where non-tech people can add/edit data.

**Comment by rijkvanzanten** [comment]
What would be an intuitive approach to enter a duration?

**Comment by santhosh-sk** [comment]
> What would be an intuitive approach to enter a duration?

Something like below (screenshot taken from [bootstrap-duration-picker](https://github.com/koss-lebedev/bootstrap-duration-picker)). The input fields could be based on min-max settings at the field level. 

<img width="361" alt="Screenshot 2020-09-18 at 18 32 47" src="https://user-images.githubusercontent.com/4300016/93622441-725ce280-f9dd-11ea-949b-0a6170d3d0cb.png">
